rma/models/rma.py

```diff
@@ -1261,6 +1258,26 @@ def _delivery_group_key(self):
         self.ensure_one()
         return (self.partner_shipping_id.id, self.company_id.id, self.warehouse_id.id)
 
+    def _reception_group_key(self):
```

Comment:
I think calling methods by this convention makes better to understand its goal (`get`: return something):

```suggestion
    def _get_reception_group_key(self):
```